flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/mysql/ParsingProcessFunction.java

```diff
@@ -62,4 +63,23 @@ public static OutputTag<String> createRecordOutputTag(String tableName) {
         return new OutputTag<String>("record-" + tableName) {
         };
     }
+
+    public String recordConvert(String record) throws Exception {
+        JsonNode recordRoot = objectMapper.readValue(record, JsonNode.class);
```

Comment:
A switch needs to be added here. Only when it is turned on does the data need to be converted.

Reply:
Can I communicate in Chinese

Reply:
> 此处需要添加一个开关。只有当它打开时，才需要转换数据。

![image](https://github.com/apache/doris-flink-connector/assets/46134044/6114ef08-1a34-4177-9148-004d71fb46d5)
![image](https://github.com/apache/doris-flink-connector/assets/46134044/da20f9f4-1af6-4878-89d7-7d7636135d13)
Actually, it has already been processed

Reply:
When the switch is turned off, there is no need to process the record, because traversing the data here may cause a loss of performance.

Reply:
> 关闭开关后，无需处理记录，因为遍历此处的数据可能会导致性能损失。

Okay, let me make some adjustments